web/app/src/App.js

```diff
@@ -13,10 +14,12 @@ class App extends Component {
                     <p>Character Card Hand: {this.props.characterCardHand}</p>
                 </div>
                 <h1>Drag & drop</h1>
-                <HexGrid width={2600} height={1000} viewBox="-50 -50 100 100">
+                <HexGrid width={1300} height={500} viewBox="-50 -50 100 100">
```

Comment:
They are changing because of the resolution of different screens. I need some discussion that which won't take much time. Sam and I will fix that.
@samridhprasad When we meet tomorrow in class we will resolve this.

Reply:
Linked #64 to address this